pkg/registry/core/service/ipallocator/allocator.go

```diff
@@ -254,7 +254,7 @@ func calculateIPOffset(base *big.Int, ip net.IP) int {
 func RangeSize(subnet *net.IPNet) int64 {
 	ones, bits := subnet.Mask.Size()
 	if (bits - ones) >= 31 {
-		panic("masks greater than 31 bits are not supported")
+		panic("masks shorter than /2 for IPv4 or /98 for IPv6 are not supported.")
```

Comment:
In my experience, subnet masks are expressed using "length". I prefer using shorter or longer with a / notation to describe a subnet mask.